lib/compat/wordpress-6.3/class-gutenberg-rest-global-styles-revisions-controller-6-3.php

```diff
@@ -282,14 +282,17 @@ public function prepare_item_for_response( $post, $request ) {
 		$fields = $this->get_fields_for_response( $request );
 		$data   = array();
 
-		if ( ! empty( $global_styles_config['styles'] ) || ! empty( $global_styles_config['settings'] ) ) {
+		if ( ! empty( $global_styles_config['styles'] ) || ! empty( $global_styles_config['settings'] ) || ! empty( $global_styles_config['behaviors'] ) ) {
```

Comment:
So because the changes in this PR won't be in Wordpres 6.3, we'll need to create a new compat folder and move the changes into there, e.g., lib/compat/wordpress-6.4/class-gutenberg-rest-global-styles-revisions-controller-6-4.php

Then we can extend 6-3, e.g., `class Gutenberg_REST_Global_Styles_Revisions_Controller_6_4 extends Gutenberg_REST_Global_Styles_Revisions_Controller_6_3 {`, making sure to move the corresponding hook that registers routes via `rest_api_init` to lib/compat/wordpress-6.4/rest-api.php